src/pods/canvas/canvas.pod.tsx

```diff
@@ -30,27 +31,27 @@ export const CanvasPod: React.FC = () => {
     newY: number,
     totalHeight: number
   ) => {
-    setSchema((prevSchema) => {
+    setSchema(prevSchema => {
       return {
         ...prevSchema,
-        tables: prevSchema.tables.map((table) => {
+        tables: prevSchema.tables.map(table => {
           if (table.id === id) {
             return {
               ...table,
               x: Math.max(0, Math.min(newX, 1200 - 300)), // Asegurar que x está dentro de los límites
-              y: Math.max(0, Math.min(newY, 800 - totalHeight)), // Asegurar que y está dentro de los límites
+              y: Math.max(0, Math.min(newY, 800 - totalHeight)) // Asegurar que y está dentro de los límites
```

Comment:
My fault, can you translate this comment into english?

Reply:
It's fixed